modules/integration_gcp-memorystore-redis/conf/03-memory-usage-ratio.yaml

```diff
@@ -0,0 +1,20 @@
+## Example
+module: gcp-memorystore-redis
+name: memory usage ratio
+
+value_unit: "%"
+transformation: ".min(over='1h')"
+
+signals:
+  signal:
+    metric: stats/memory/system_memory_usage_ratio
```

Comment:
I think you mean metric `stats/memory/usage_ratio`.

[List of all available metrics.](https://cloud.google.com/memorystore/docs/redis/monitoring-metrics)